jupyterlab_automl/jupyterlab_automl/handlers.py

```diff
@@ -102,11 +148,19 @@ def _project(_):
 @_handler("POST", "createTablesDataset")
 def _create_tables_dataset(args):
   file_source = args.get("fileSource")
+  df_source = args.get("dfSource")
   if file_source:
+    decoded = base64.decodebytes(file_source["data"])
     AutoMLService.get().create_dataset_from_file(
         display_name=args["displayName"],
         file_name=file_source["name"],
-        file_data=file_source["data"])
+        file_data=decoded)
+  elif df_source:
+    decoded = base64.decodebytes(df_source)
+    AutoMLService.get().create_dataset_from_file(
+        display_name=args["displayName"],
+        file_name=args["displayName"],
+        file_data=decoded)
```

Comment:
I'm pretty sure this can be removed since we have `create_dataset_from_dataframe`